client/my-sites/site-settings/podcasting-details/style.scss

```diff
@@ -1,3 +1,87 @@
+// Podcasting section/link on Writing settings page
+
+.podcasting-details__link {
+	&:not( .is-enabled ) {
+		.podcasting-details__link-button {
+			padding-left: 24px;
+			padding-right: 24px;
+		}
+	}
+
+	.podcasting-details__link-action-container {
+		display: flex;
+		align-items: flex-start;
+
+		.podcasting-details__link-info {
+			flex-grow: 1;
+			color: $gray-text;
+			margin-right: 12px;
+		}
+		.podcasting-details__link-button {
+			color: $gray-darken-20;
+			font-weight: bold;
+			flex-shrink: 0;
+		}
+	}
+
+	a.external-link {
+		text-decoration: underline;
+		white-space: nowrap;
+
+		&:hover, &:focus, &:active {
+			color: $link-highlight;
+		}
+
+		.gridicons-external {
+			margin-left: 5px;
+			color: $gray-text-min;
+		}
+	}
+
+	.podcasting-details__link-feed-label {
+		font-weight: 600;
+		margin-bottom: 5px;
+	}
+}
+
+.podcasting-details__link.is-enabled {
+	.podcasting-details__link-info {
```

Comment:
Adding the `border-radius: 4px` from the mockup helps to make this not as sharp.